docs/monitoring/spm-faq.md

```diff
@@ -1063,16 +1063,16 @@ Note:
 Usage is metered hourly on a per-agent basis. For example:
 
 If you send metrics from a server A to Monitoring App Foo between 01:00 and
-02:00 that's $0.035 for the Standard plan.
+02:00 that's $0.005 for the Standard plan.
 
 If another agent is monitoring something else, even if that is running on the same
-server A, and sending metrics to a different Monitoring App Bar, that's another $0.035.
+server A, and sending metrics to a different Monitoring App Bar, that's another $0.005.
 
 If you are not sending metrics from a server A for a Monitoring App Foo between
 02:00 and 03:00 then you pay $0 for that hour.
 
```

Comment:
@cpipilas Below 2 lines are correct for *Infra* (but not Integrations) monitoring.

Reply:
Updated to include both Infra & Integrations.